ravenframework/Optimizers/mutators/mutators.py

```diff
@@ -73,7 +72,11 @@ def scrambleMutator(offSprings, distDict, **kwargs):
           variables, list, variables names.
     @ Out, child, np.array, the mutated chromosome, i.e., the child.
   """
-  loc1,loc2 = locationsGenerator(offSprings, kwargs['locs'])
+  if kwargs['locs'] is None:
+    locs = list(set(randomUtils.randomChoice(list(np.arange(offSprings.data.shape[1])),size=2,replace=False)))
+  else:
+    locs = [kwargs['locs'][0], kwargs['locs'][1]]
```

Comment:
why locationGenerator can not be used here?

Reply:
It can be, fixed.